src/addandremove.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+
+import {
+  //   clearAll,
+  form,
+  arr,
+  editFormArr,
+} from './index.js';
+
+export function addTodo() {
+  form.addEventListener('submit', () => {
+    const input = document.querySelector('.addlist');
+
+    const toDo = input.value;
+    if (toDo) {
+      arr.push({
+        name: toDo,
+        completed: false,
+        index: arr.length + 1,
+      });
+
+      localStorage.setItem('List', JSON.stringify(arr));
+    }
+    input.value = '';
+  });
+}
+
+export function editTask() {
+  editFormArr.forEach((form) => {
+    form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      const { value } = form.querySelector('.text');
+      const id = Number(form.querySelector('.text').id);
+      const newArr = JSON.parse(localStorage.getItem('List')) || [];
+      newArr.forEach((item) => {
+        if (item.index === id) {
+          item.name = value;
+          localStorage.setItem('List', JSON.stringify(newArr));
+          location.reload();
+        }
+      });
+    });
+  });
+}
+
+export function removeElement(task, trash) {
```

Comment:
- [ ] In this activity we want your app to work in a way that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. When I enter an item _one_ and it has an `index of 1`, and then adding an item _two_ is also assigned an `index of 2`, which is great! However, when I delete item _one_, item two should now be assigned an `index property of 1`. Let's improve upon this aspect.

**Mismatch in index property after deleting an item:**
![image](https://user-images.githubusercontent.com/12745474/221358795-4467ec4a-ff18-49c5-ad59-2e869ef098a1.png)

Reply:
- [ ] You have not implemented the change requested by te previous reviewer. `Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..` This is what happens when I delete a task  😸 
![index](https://user-images.githubusercontent.com/99656687/221361053-65250118-548f-45a0-a55c-576be2a11a76.png)
Please note that this is a project requirement highlighted in red below
![pic](https://user-images.githubusercontent.com/99656687/221361083-e119ded3-5728-47af-8773-1fe293436cf6.png)


- [OPTIONAL] `By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).` It would be great if you implemented this feature too so that the tasks are indexed from 1.. and not 0. This is highlighted in green in the tasks above. 👏🏾